### PR TITLE
[wip] Add fakechroot to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
Sometimes we need to interact with OS level packages that have not been packaged into conda, but can be forced to install in a "chrooted" environment. Since most times, conda users don't have root access `fakechroot` to the rescue.

http://github.com/dex4er/fakechroot

### CODE REVIEW

**Comment by sodre** [comment]
@isuruf, this is the last one that needs review in my queue :) 

Thank you so much for taking the time to do this!

File: recipes/fakechroot/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+./autogen.sh
+./configure --prefix $PREFIX
+
+make -j $CPU_COUNT
+set +e
```
Comment (isuruf) [inline recipes/fakechroot/build.sh]: Why?

Comment (sodre) [reply recipes/fakechroot/build.sh]: I wanted to run the tests, but it looks like they can only run successfully as the root user.

Let me investigate this further and I'll ping you again. I think I need the "fake root" library to actually run the tests correctly.

**Comment by sodre** [comment]
Investigate using "pseudo" instead of fakechroot et.al

**Pull request closed without merging**